07-routing-gossip.md

```diff
@@ -445,6 +445,7 @@ The origin node:
   `announce_channel` bit was not set).
     - MUST NOT forward such a `channel_update` to other peers, for privacy
     reasons.
+    - MUST NOT send a created `channel_update` before `funding_locked` has been received.
```

Comment:
I'm not sure this is the right place to put this. Right now this is a sub-case of when we want to create a `channel_update` _before_ announcing the channel.
I think the restriction you want to add should apply to all `channel_updates`, so it should be on a different line shouldn't it? Maybe before line 443 for example.

Reply:
Thank you.
Move to before L.443.